### ISSUE TITLE
Pixel 7 Pro and TicWatch Atlas

### ISSUE THREAD

**Issue by EmerJency** [Silleellie/dnd-bedtime-sync#15]
Hey love the project. There are a couple things that don't sync for me either way. Bedtime Mode doesn't sync nor does low power mode. DND does sync both ways. I do have the wellness app on my pixel. Hope you still get these reports. 

EJ

**Comment by Turtlepaw** [comment]
@EmerJency low power mode cannot be changed by user apps without root, but Bedtime Mode automatically makes the device go into a deep sleep, which actually saves more battery than battery saver.

**Comment by EmerJency** [comment]
Hey @Turtlepaw thanks for the reply. I actually tried your branch of DNDsync v3.0, but couldn't get phone synced with watch when I hit DND on watch. It did sync from phone to watch. Again, bedtime Mode doesn't sync either way and low power mode on watch doesn't turn on when DND is initiated. 

Happy New Year!
EJ

**Comment by Turtlepaw** [comment]
> Hey @Turtlepaw thanks for the reply. I actually tried your branch of DNDsync v3.0, but couldn't get phone synced with watch when I hit DND on watch. It did sync from phone to watch. Again, bedtime Mode doesn't sync either way and low power mode on watch doesn't turn on when DND is initiated.
> 
> Happy New Year! EJ

Thanks for reporting these bugs @EmerJency, it should hopefully be fixed in [3.0.1](https://github.com/Turtlepaw/dnd-bedtime-sync/releases/tag/3.0.1), it was an issue where preferences were [not correctly written](https://github.com/Turtlepaw/dnd-bedtime-sync/commit/e4b7f22cd9197e71c6388244079542f46184b218#diff-8134b9bd0d8de54ea72b737c211ecd62f8ea7cb28fda497b2a278882b5f6d155L20).

Happy new year! 🎉

**Comment by EmerJency** [comment]
Wow @Turtlepaw, that was quick.  I just tested v3.01 and all seems to work except for power saving to watch.  Does my phone need to be rooted for that to work?  Thanks for the super quick update, and keep up the great work!

EJ

**Comment by Turtlepaw** [comment]
> Wow @Turtlepaw, that was quick. I just tested v3.01 and all seems to work except for power saving to watch. Does my phone need to be rooted for that to work? Thanks for the super quick update, and keep up the great work!
> 
> EJ

@EmerJency, unfortunately, only system apps are allowed to change the battery saver mode, but it is possible to change it on rooted devices ([source](https://stackoverflow.com/a/31369841/15751555)), but you would have to root your *watch* for that to work.

Although, as I've said before, Bedtime Mode already puts your watch in a very deep sleep, so enabling battery saver along with Bedtime Mode is unnecessary.

**Comment by EmerJency** [comment]
@Turtlepaw Yes, I get it. I just thought that since the option was there to enable power saving mode with bedtime mode I could benefit from it. Maybe remove the option if it is redundant?  Or label it as a rooted device option?  Not meaning to be critical, just trying to contribute what I can. 

EJ

**Comment by Turtlepaw** [comment]
> @Turtlepaw Yes, I get it. I just thought that since the option was there to enable power saving mode with bedtime mode I could benefit from it. Maybe remove the option if it is redundant?  Or label it as a rooted device option?  Not meaning to be critical, just trying to contribute what I can. 
> 
> EJ

There's not many rooted Wear OS devices, so I'll likely just remove it. Since it was (potentially?) working on @Silleellie's device, I just kept it there assuming it worked fine on Galaxy devices.

**Comment by EmerJency** [comment]
Well, I'll be watching for the next update. 😁 For my part, it does what I need it to. Thanks again for the quick updates and again Happy New Year. 

EJ

**Comment by Turtlepaw** [comment]
> Well, I'll be watching for the next update. 😁 For my part, it does what I need it to. Thanks again for the quick updates and again Happy New Year. 
> 
> EJ

@EmerJency btw, if you're looking for more awesome Wear OS apps, keep an eye on [awesome-wear-os](https://github.com/WearOSCommunity/awesome-wear-os), I'm going to be adding more awesome apps and feel free to open a PR if you found any more awesome apps for Wear OS 🙂
